src/sa_web/static/js/views/dataset-form-view.js

```diff
@@ -0,0 +1,198 @@
+/*globals _ Spinner Handlebars Backbone jQuery Gatekeeper */
+
+var Shareabouts = Shareabouts || {};
+
+(function(S, $, console){
+  S.DatasetFormView = S.PlaceFormView.extend({
+    events: {
+      'submit form': 'onSubmit',
+      'change input[type="file"]': 'onInputFileChange',
+      'click .category-btn-label-clickable': 'onCategoryChange',
+      'click .category-menu-hamburger': 'onExpandCategories',
+      'click input[data-input-type="binary_toggle"]': 'onBinaryToggle'
+    },
+    initialize: function(){
+      var self = this;
+      // keep track of relevant catgory & dataset info as user switches among categories
+      this.selectedCategory = null;
+      this.selectedDatasetId = null;
+      this.priorDatasetId = null;
+      this.selectedDatasetSlug = null;
+      this.priorModelCid = null;
+      S.TemplateHelpers.overridePlaceTypeConfig(this.options.placeConfig.items,
+        this.options.defaultPlaceTypeName);
+      S.TemplateHelpers.insertInputTypeFlags(this.options.placeConfig.items);
+
+      // attach collection listeners
+      for (var collection in this.collection) {
+        this.collection[collection].on('add', self.setModel, this);
+      }
+    },
+    render: function(category, category_selected){
+      // Augment the model data with place types for the drop down
+      
+      // This is a little hacky--I need to find a better way to extend the template helpers
+      // One option is to stop relying on them entirely and just use registered Handlebar helper functions
+      if (category != undefined) {
+        S.TemplateHelpers.insertInputTypeFlags(this.options.placeConfig.categories[category].fields);
+      }
+
+      var data = _.extend({
+        place_config: this.options.placeConfig,
+        selected_category: this.options.placeConfig.categories[category],
+        category_selected: category_selected || false,
+        user_token: this.options.userToken,
+        current_user: S.currentUser
+      }, S.stickyFieldValues);
+
+      this.$el.html(Handlebars.templates['dataset-form'](data));
+
+      // initialize datetime picker, if relevant
+      $('#datetimepicker').datetimepicker({ formatTime: 'g:i a' });
+
+      return this;
+    },
+    getAttrs: function() {
+      var attrs = {},
+          locationAttr = this.options.placeConfig.location_item_name,
+          $form = this.$('form');
+
+      // Get values from the form
+      attrs = S.Util.getAttrs($form);
+
+      // get values off of binary toggle buttons that have not been toggled
+      $.each($("input[data-input-type='binary_toggle']:not(:checked)"), function() {
+        attrs[$(this).attr("name")] = $(this).val();
+      });
+
+      // Get the location attributes from the map
+      attrs.geometry = {
+        type: 'Point',
+        coordinates: [this.center.lng, this.center.lat]
+      };
+
+      if (this.location && locationAttr) {
+        attrs[locationAttr] = this.location;
+      }
+
+      console.log("attrs", attrs);
```

Comment:
We should remove unneeded console statements. Although it might be helpful to log some information, but if we do, we should use the actual logger from our utilities